src/Spectre.Data/Spectre.Data.csproj

```diff
@@ -34,8 +34,32 @@
     <DocumentationFile>bin\x64\Release\Spectre.Data.xml</DocumentationFile>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Web.Infrastructure, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Could we find a way get rid of at least some of these dependencies?

Reply:
Honestly I don't know what to do about that. I'll ask colleagues and stack overflow.

Reply:
Well, that what has been bothering me. Lots of MVC-related dependencies but I was like "m'kay, if we need this in order to do it properly and in future-safe way, so be it".